File: 1-Iniciante/ex1012.py
#entrada
a, b, c = input().split()
pi = float(3.14159)

#processamento/saída
a = float(a)
b = float(b)
c = float(c)

print("TRIANGULO: {:.3f}".format((a * c) / 2))
print("CIRCULO: {:.3f}".format((pi * c**2)))
print("TRAPEZIO: {:.3f}".format(((a + b) * c)/2))
print("QUADRADO: {:.3f}".format(b**2))
print("RETANGULO: {:.3f}".format(a * b))

File: 1-Iniciante/ex1012.js
// entrada
let input = require('fs').readFileSync('dev/stdin', 'utf-8'); //adicionar uma barra antes do 'dev' para funcionar no beecrowd
let [a, b, c] = input.split(' ').map(n => parseFloat(n));

// processamento
let tri = (a * c) / 2;
let cir = 3.14159 * c ** 2;
let tra = ((a + b) * c) / 2;
let sqr = b ** 2;
let rec = a * b;

// saida
console.log('TRIANGULO: ' + tri.toFixed(3));
console.log('CIRCULO: ' + cir.toFixed(3));
console.log('TRAPEZIO: ' + tra.toFixed(3));
console.log('QUADRADO: ' + sqr.toFixed(3));
console.log('RETANGULO: ' + rec.toFixed(3));